tools/python/clean_strings_txt.py

```diff
@@ -340,6 +355,7 @@ def find_omim():
 
 
 def read_hardcoded_categories(a_path):
+    logging.info("Loading harcoded categories from: {0}".format(a_path))
```

Comment:
What is not cool is that `data/hardcoded_categories.txt` is actually not used by the code, but the contents are duplicated in `search/displayed_categories.cpp` - it should be loaded from the file instead.

Reply:
That problem was introduced by the author of the python script. The proper approach should be to delete data/hardcoded_categories.txt and to parse values from the cpp file.

Reply:
Hmm, I see.
Though having the categories not hardcoded, but in an editable resource file is overall a better approach, isn't it?

Reply:
The ideal solution would be to load them from categories.txt directly (but some custom starting UI order should be also supported, which can change dynamically later when a user uses something more often).

But using a separate file, unsynced with the code, just for scripts, is definitely the wrong way.

Reply:
Ok, we'll get to it sometime
#1795

Reply:
Please insert todo here about reading from cpp file.

Reply:
Done!